src/wallet/rpc/wallet.cpp

```diff
@@ -234,8 +234,21 @@ static RPCHelpMan loadwallet()
                     }
                 },
                 RPCExamples{
-                    HelpExampleCli("loadwallet", "\"test.dat\"")
-            + HelpExampleRpc("loadwallet", "\"test.dat\"")
+                    "\nLoad wallet with files under wallets/foo/:\n"
+                    + HelpExampleCli("loadwallet", "\"foo\"")
+                    + HelpExampleRpc("loadwallet", "\"foo\"")
+                    + "\nLoad wallet with files under wallets/foo/specialWallet/:\n"
+                    + HelpExampleCli("loadwallet", "\"foo/specialWallet/\"")
+                    + HelpExampleRpc("loadwallet", "\"foo/specialWallet/\"")
+                    + "\nLoad wallet using absolute path (Unix):\n"
+                    + HelpExampleCli("loadwallet", "\"/path/to/specialWallet/\"")
```

Comment:
```suggestion
                    + HelpExampleCli("loadwallet", "\"/path/to/walletname/\"")
```
more clear imo
same for other ones

Reply:
Done!